### PR TITLE
Buider with required field for PreparedPutContentValues

### PR DESCRIPTION
**Pull request by nikitin-da**
@artem-zinnatullin Check, please

### CODE REVIEW

**Comment by nikitin-da** [comment]
Sorry, on laptop tests take more then 10 minutes =( 
I will fix it soon.

**Comment by nikitin-da** [comment]
tests are works!

**Comment by artem-zinnatullin** [comment]
Fix: `tests work` or `tests are working`

Great, btw take a look at https://github.com/google/dagger/pull/157#discussion_r28016740 I think we need CLA Bot for Pushtorefresh which will ask us to sign CLA on every PR :)

File: storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java [hunk 1]
```diff
@@ -92,8 +94,30 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull Co
...
+        public CompleteBuilder withPutResolver(@NonNull PutResolver<ContentValues> putResolver) {
             this.putResolver = putResolver;
+            return new CompleteBuilder(this);
+        }
+    }
+
+    /**
+     * Builder for {@link PreparedPutContentValues}
+     */
+    public static class CompleteBuilder extends Builder {
```
Comment (artem-zinnatullin) [inline storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java]: I think, `CompleteBuilder` should override all `Builder` methods and change return type to `CompleteBuilder`?

Comment (artem-zinnatullin) [inline storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java]: Oh, sorry, I thought there are other params in the Builder, everything okay

**Comment by artem-zinnatullin** [comment]
Thanks!

**Comment by nikitin-da** [comment]
tests do work

**Comment by artem-zinnatullin** [comment]
Refs #218

**Pull request merged**